### ISSUE TITLE
Other extensions?

### ISSUE THREAD

**Issue by JuMalIO** [pimusicbox/mopidy-musicbox-webclient#263]
Is it possible to add another url link, similar like its done with "alarmclock" link?
I think it should be more customizable, maybe someone else will want to add another.

**Comment by kingosticks** [comment]
It should be quite easy to edit the source files (you can find the location of them by running `mopidy deps`). I don't want to add a config option just for this.

**Comment by JuMalIO** [comment]
Yes, I did, but still think it would be nice to have custom extension links. I guess its feature request.

My current hackish solution is editing index file:
`nano /usr/local/lib/python2.7/dist-packages/mopidy_musicbox_webclient/static/index.html`
change data-has-alarmclock="true":
`<body data-websocket-url="{{ websocketUrl }}" data-is-musicbox="{{ isMusicBox }}" **data-has-alarmclock="true" data-on-track-click="{{ onTrackClick }}" data-program-name="{{ programName }}" data-hostname="{{ hostname }}" data-title="{{ title }}">`
 and change links:
`<a href="/your_link_instead_of_alarmclock/">`
`<a id="homeAlarmClock" href="/your_link_instead_of_alarmclock/">`
